.github/workflows/pip-release.yml

```diff
@@ -64,7 +64,7 @@ jobs:
     runs-on: windows-latest
     strategy:
       matrix:
-        target: [x64, x86]
+        target: [x64]
```

Comment:
So we don't build for 32-bit `x86` systems anymore? If so, we should mention explicitly in the changelog and release notes that we removed support for this target.

Reply:
Yes, so, basically, I had doucle checked our cli release and we never released a i386 windows version so, we never truly supported i386 which is x86 target on maturin. 

I removed it so that it's more specific. 

I don't know why https://github.com/dora-rs/dora/actions/runs/9693826084 is failing and as we don't support it cli side as well I just simply removed this.

Reply:
I have since then manually pushed version into pip for 0.3.5